backup/queries_table_defs.go

```diff
@@ -667,19 +673,27 @@ func GetTableInheritance(connectionPool *dbconn.DBConn, tables []Relation) map[u
 		}
 		// If we are filtering on tables, we only want to record dependencies on other tables in the list
 		if len(tableOidList) > 0 {
-			tableFilterStr = fmt.Sprintf("\nAND i.inhrelid IN (%s)", strings.Join(tableOidList, ","))
+			tableFilterStr = fmt.Sprintf("\nWHERE i.inhrelid IN (%s)", strings.Join(tableOidList, ","))
 		}
 	}
 
+	if connectionPool.Version.Before("7") {
```

Comment:
I miss a comment which explains this new branch and it seems that tableFilterStr got new semantic.
Probably, it's better to create new variable to store ExtensionFilterClause("p") value.

Reply:
fixed